### PR TITLE
HTTP module of uws (0.13)

### PR DESCRIPTION
**Pull request by jucrouzet**
- [X] Make your PR against the `master` branch.
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `tsc` without errors.
- [X] Run `npm run lint package-name` if a `tslint.json` is present. (Errors were present but did not introduced new ones)

- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/uWebSockets/uWebSockets/wiki/(More-than)-one-million-requests-per-second-in-Node.js
- [X] Increase the version number in the header if appropriate.

### CODE REVIEW

File: uws/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
-// Type definitions for µWS
+// Type definitions for µWS 0.13
```
Comment (anubmat) [inline uws/index.d.ts]: @jucrouzet , can you change Greek char mu to 'u' in uWS

Comment (jucrouzet) [inline uws/index.d.ts]: Done

**Comment by jucrouzet** [comment]
Hi @plantain-00,

@dt-bot didn't passed on the first commit (due to a header error). Do you have time to look at this PR ?

**Comment by plantain-00** [comment]
👍 thanks

**Comment by jucrouzet** [comment]
:champagne: Is it possible to merge it ?

**Comment by jucrouzet** [comment]
Hello @plantain-00 

Any chance to get it merged ? Thanks !

**Comment by plantain-00** [comment]
@jucrouzet I'm not a member, I can't merge it.
You can try to @ some active members, eg: @minestarks can you review this?

**Pull request merged**